salt/metalk8s/orchestrate/deploy_node.sls

```diff
@@ -40,31 +44,46 @@ Refresh the mine:
     - name: mine.update
     - tgt: '*'
 
+Cordon the node:
+  metalk8s_kubernetes.node_cordoned:
+    - name: {{ node_name }}
+    - kubeconfig: {{ kubeconfig }}
+    - context: {{ context }}
+
+# TODO: Call drain
+
 Run the highstate:
   salt.state:
-    - tgt: {{ pillar['node_name'] }}
+    - tgt: {{ node_name }}
     - highstate: True
     - require:
       - salt: Set grains
       - salt: Refresh the mine
+      - metalk8s_kubernetes: Cordon the node
+
+Uncordon the node:
+  metalk8s_kubernetes.node_uncordoned:
+    - name: {{ node_name }}
+    - kubeconfig: {{ kubeconfig }}
+    - context: {{ context }}
```

Comment:
Wouldn't this `require` `Run the highstate`?

Reply:
True, I just forgot this